.sample.env

```diff
@@ -6,3 +6,5 @@ OUTBOUND_EMAIL_DOMAIN=staging.constable.io
 PERMITTED_EMAIL_DOMAIN=thoughtbot.com
 SHUBOX_SCRIPT_URL=get-from-shubox-dashboard
 SLACK_WEBHOOK_URL=https://fakeslack/webhook
+HUB_URL=https://staging-hub.thoughtbot.com
+HUB_API_TOKEN=get-value-from-hub-staging
```

Comment:
What do you think about adding the env param name to this placeholder value?

E.g., `get-API_TOKEN-value-from-thoughtbot-hub-staging`

I wasnt sure what value this needed to be when I first glanced at this file. I would intuitively guess that Hub would issue Constable its own API key, so that Id need to look in Constable-staging, rather than Hub-staging. I then looked at the Heroku settings page for Hub-staging, and it felt obvious to me that the value for `API_TOKEN` is what I need to select. Hub also has a lot of other env params. Im wondering if anyone else would pause for a moment over this like I did.